### PR TITLE
[GH-1823] Added E2E test for MM-T4284

### PR DESCRIPTION
**Pull request by venarius**
#### Summary
Hey, everyone! This is my first commit. I would be grateful for any feedback!

This commit adds a cypress E2E test for MM-T4284

#### Ticket Link

Fixes https://github.com/mattermost/focalboard/issues/1823

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by saturninoabril — CHANGES_REQUESTED** [review]
@venarius Thank you for your contribution! Could you please update to latest main since there's a recent commit that changed basic Cypress setup?
I'll look into the details once updated. Also, please share a screenshot that the test is passing on your local dev.

**Comment by venarius** [comment]
@saturninoabril 
image

File: webapp/cypress.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "baseUrl": "http://localhost:8088",
     "testFiles": [
         "**/login*.ts",
-        "**/create*.ts"
+        "**/create*.ts",
+        "**/manage*.ts"
```
Comment (venarius) [inline webapp/cypress.json:7]: This seems somewhat weird to me. Why not suffix all cypress integration files with `.spec.ts` so we could do `"**/*.spec.ts"` here instead?

**Review by hahmadia — APPROVED** [review]

File: webapp/cypress.json (hunk 1 above)
Comment (saturninoabril) [reply webapp/cypress.json:7]: I agree. It's done this way to maintain order but we'll change it next time for a better approach.

**Review by saturninoabril — APPROVED** [review]
Thanks @venarius! LGTM.

**Pull request merged**